easybuild/easyconfigs/p/PETSc/PETSc-3.22.5-foss-2023b.eb

```
@@ -0,0 +1,45 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+# Author:    Jasper Grimm (UoY)
+##
+name = 'PETSc'
+version = '3.22.5'
+
+homepage = 'https://www.mcs.anl.gov/petsc'
+description = """PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the
+ scalable (parallel) solution of scientific applications modeled by partial differential equations."""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+toolchainopts = {'openmp': True, 'usempi': True, 'pic': True}
+
+source_urls = [
+    'https://web.cels.anl.gov/projects/petsc/download/release-snapshots',
+]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['984dba48bd26e7b17d42c078fc4f74d59e9cbc437ee25a8635865eeca9f5dd28']
+
+builddependencies = [('CMake', '3.27.6')]
+
+dependencies = [
+    ('Python', '3.11.5'),
+    ('SciPy-bundle', '2023.11'),
+    ('Boost', '1.83.0'),
+    ('METIS', '5.1.0'),
+    ('SCOTCH', '7.0.4'),
+    ('MUMPS', '5.6.1', '-metis'),
+    ('SuiteSparse', '7.7.0', '-METIS-5.1.0'),
```

The `versionsuffix` here should probably be dropped (just like it is in the existing `PETSc-3.20.3-foss-2023a.eb`):

```suggestion
    ('SuiteSparse', '7.7.0'),
```

---

Indeed, the `versionsuffix` was dropped in #21199, so we should change this accordingly here for the dependency to resolve